server/lib/security/expense.ts

```diff
@@ -42,6 +42,15 @@ const getExpensesStats = where =>
 const addBooleanCheck = (checks, condition: boolean, ifTrue: SecurityCheck, ifFalse?: SecurityCheck) =>
   condition ? checks.push(ifTrue) : ifFalse ? checks.push(ifFalse) : null;
 
+const getTimeWindowFromDate = (
+  date: moment.MomentInput,
+  amount: moment.DurationInputArg1,
+  unit: moment.DurationInputArg2,
+) => ({
+  [Op.gte]: moment(date).subtract(amount, unit).toDate(),
```

Comment:
Not in scope of this PR, but just wanted to note I think `moment` is deprecated now? Not sure if we have a current transition plan towards something like `luxon` or similar.

Reply:
Not sure, currently being tracked here: https://github.com/opencollective/opencollective/issues/4524